client/src/modules/entities/entities.service.js

```diff
@@ -0,0 +1,32 @@
+angular.module('bhima.services')
+  .service('EntityService', EntityService);
+
+EntityService.$inject = ['PrototypeApiService'];
+
+/**
+ * @class EntityService
+ * @extends PrototypeApiService
+ *
+ * @description
+ * Encapsulates common requests to the /entities/ URL.
+ */
+function EntityService(Api) {
+  const baseUrl = '/entities/';
+  const service = new Api(baseUrl);
+  service.types = new Api(baseUrl.concat('types/'));
+
+  // clean entity
+  service.clean = entity => {
+    const valid = {};
```

Comment:
Did you know you can do this with [`util.maskObjectFromKeys()`](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/client/src/js/services/util.js#L228)?  Like this:
```js
service.clean = entity => util.maskObjectFromKeys(entity, ['uuid', 'display_name', 'gender', 'email', 'phone', 'address', 'entity_type_id']);
```
.. or even more compact:
```js
const entityKeys = ['uuid', 'display_name', 'gender', 'email', 'phone', 'address', 'entity_type_id'];

service.clean = entity => util.maskObjectFromKeys(entity, entityKeys);
```